ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -12,11 +12,18 @@
 # Define connected endpoints keys, to define grouping of endpoints connecting to the fabric.
 # This provides the ability to define various keys of your choice to better organize/group your data.
 # This should be defined in top level group_var for the fabric.
+# Lookup source can be used to read data directly from an external source. Even with regular YAML files, this can speed up ansible runtime considerably.
 connected_endpoints_keys:
   < key_1 >:
     type: < type used for documentation >
   < key_2 >:
     type: < type used for documentation >
+    source: < "inventory" | "lookup" | default -> "inventory" >
```

Comment:
It might be useful to pull from multiple sources, if data is in multiple systems or to help transition from one system to another. Same for network services.

Data model suggestion:
```yaml
   sources:
     - name: ansible_inventory
        type: inventory (default)
     - name:  < descriptive name source > 
        type: lookup  
        lookup:
          name: < name_of_lookup_plugin >
          arguments: [ < lookup_plugin_argument1 >, < lookup_plugin_argument2 > ]
          errors: < "strict" | "warn" | "ignore" | default -> "strict" >
          data_filter: < "from_yaml" | "from_json" >
```

Reply:
We can have multiple keys, so just add another key to pull from another source.